packages/block-library/src/group/edit.js

```diff
@@ -43,6 +68,26 @@ function GroupEdit( {
 						},
 					] }
 				/>
+
+				<PanelBody title={ __( 'Spacing' ) }>
+					<ResponsiveBlockControl
+						legend="Padding"
+						property="padding"
+						isOpen={ showResponsiveControls }
+						onToggle={ ( isOn ) => {
+							setState( {
+								showResponsiveControls: ! isOn,
+							} );
+						} }
+						renderDefaultControl={ ( label ) => (
+							<SelectControl
```

Comment:
Unless there's some other magic taking care of it, I'm missing an `onChange` here. Is this actually saving the settings per-device?

Reply:
> This isn't in any way wired up to Block state...

I should have read the full description first, sorry 🤦‍♂

Reply:
Please see PR description.